### DISCUSSION TITLE
issue using plotman interactive: "curses.error: addwstr()" error

### DISCUSSION THREAD

**Discussion by username4m** [ericaltendorf/plotman#352] [General]
i was able to set up the config file fine and plotman appears to be plotting correctly.  when i use plotman status it shows a plot being created.  i attempted to use plotman interactive and received this error
![unknown](https://user-images.githubusercontent.com/84050348/117897799-20790d80-b289-11eb-948d-bbe7cd17f607.png)

**Comment by BasilHorowt** [comment]
@username4m your terminal is too small.  I've added this question to the FAQ https://github.com/ericaltendorf/plotman/wiki/FAQ

**Reply by username4m** [reply]
sorry im new to Linux and dont want to show any sensitive info (though i dont think any is present)  ill post the full error if that would help

**Reply by BasilHorowt** [reply]
@username4m sorry, I meant: this error is caused by your terminal being too small :)

**Reply by username4m** [reply]
wow.  thats how ignorant to linux i am!  thanks brother.  all i had to do was get a bigger window lol

**Reply by username4m** [reply]
sence you are here, can you point me to a write up on how to set my config to do more plots at once?  i have a 5950x and 4tb ssd and can do a few in parallel i assume

**Reply by BasilHorowt** [reply]
haha no prob, it is a funny one. I don't think you will encounter that error very frequently: it's a detail of the UI library Plotman uses for interactive.  I think there is a fix to give a more helpful error message but it must not be in the version you installed yet

**Reply by altendky** [reply]
Yeah, that's not Linux ignorance.  It is a confusing error message that simply shouldn't happen to begin with.  There's work in progress towards that end.  v0.3.1 hopefully will at least suggest making your terminal bigger when this happens.
